README.md

```diff
@@ -0,0 +1,24 @@
+# message-handling [![Build Status](https://github.com/InformatieVlaanderen/message-handling/workflows/CI/badge.svg)](https://github.com/Informatievlaanderen/message-handling/actions)
```

Comment:
👍 always nice to have a readme!

Reply:
Read me still incomplete. But will continue writing it between 🥣 & 🥔